packages/itwin/ec3-widget/src/components/EC3/EC3Config.ts

```diff
@@ -0,0 +1,30 @@
+import type { GetAccessTokenFn } from "../api/context/ApiConfigContext";
+
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+export class EC3Config {
+  public readonly clientId: string;
+  public readonly scope = "read%20write";
+  public readonly redirectUri: string;
+  public readonly ec3Uri: string;
+  public readonly prefix: "dev" | "qa" | undefined;
+  public readonly getAccessToken?: GetAccessTokenFn;
+
+  constructor(props: EC3ConfigProps) {
+    this.clientId = props.clientId;
+    this.redirectUri = props.redirectUri;
+    this.ec3Uri = props.ec3Uri ?? "https://buildingtransparency.org/";
+    this.prefix = props.prefix;
+    this.getAccessToken = props.getAccessToken;
```

Comment:
If `getAccessToken` is not provided some default should be used. If this class is handling defaults, then the substitution should be done here.

Reply:
added default ```getAccessToken``` handling in ```EC3Config```

Reply:
resolved